data/darktableconfig.xml.in

```diff
@@ -566,12 +566,40 @@
     <longdescription>sort the following collections in descending order: 'film roll' by folder, 'folder', 'times' (e.g. 'date taken')</longdescription>
   </dtconfig>
   <dtconfig>
-    <name>ui_last/colorpicker_mean</name>
+    <name>ui_last/colorpicker_model</name>
+    <type>int</type>
```

Comment:
Don't know if defining this as enum, like other examples in this file, would work here?
 <type>
      <enum>
        <option>RGB</option>
        <option>Lab</option>
        etc
      </enum>

Reply:
Yes, it should be! Both `ui_last/colorpicker_model` and `ui_last/colorpicker_mode`. After all this rewriting, I didn't want to add one more wrinkle and change that! But you're right, it's probably a good idea to get that right as well...

Reply:
OK, I did this! It's certainly the more proper way to do it.